### ISSUE TITLE
[Bug]: ALT button has low contrast in pink theme

### ISSUE THREAD

**Issue by Skivling** [pixelfed/pixelfed-rn#338]
### Description

In the pink theme, the ALT text button is dark pink on black which is low contrast. The other themes are high contrast with white on black. Ideally it should be a lighter pink.

### Steps to reproduce

1. switch to pink theme
2. turn on alt text button
3. see that the contrast is low on the button

### Pixelfed version

1.2.0.1

### Platforms

Android

### Device model

N/A

### Acknowledgements

Yes

**Comment by ymau** [comment]
In the light theme ithe button has black text on a black background.

**Comment by Skivling** [comment]
@ymau that must be a recent bug, a few days ago it was white text on black background.

**Comment by ymau** [comment]
Its **v1.2.0.1** on **Android** (in my case: Android 12 on a Xiaomi Poco F3 with MIUI 13.0.7)

**Comment by Skivling** [comment]
I have v1.2.0.1 as well, must have mistyped my original comment. (fixed original)
